packages/sdk-client-v3/src/client/client.ts

```diff
@@ -177,7 +177,7 @@ export default function createClient(middlewares: ClientOptions): Client {
         })
           .then((res) => {
             if (res.error) {
-              reject(res.error)
+              reject(res)
```

Comment:
Why did you have to do this?

Reply:
https://github.com/commercetools/commercetools-sdk-typescript/pull/708/commits/0f40d0dd0110fd858a91073190e8a509772c3c05#diff-9ec95fcfeee72e5f31f857b91ea4a3a887925d7643174db88b0e12761a764a8dR180